[DI-6981] Update documentation for anonymous users in Segment (#8733)
braze-inc/braze-docs: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/segment/segment_for_currents.md

2. Click **+ Create New Current** > **Segment Data Export**.
3. Next, provide an integration name, contact email, Segment write key, and Segment region.

![The Segment Currents page in Braze. Here, you can find fields for integration name, contact email, segment region, and API key.][1]

### Step 3: Export message engagement events

> Next, select the message engagement events you would like to export. Reference the following export events and properties table listed. All events sent to Segment will include the user's `external_user_id` as the `userId` and the user's `braze_id` as the `anonymousId`. Braze will only send event data for users who do not have their `external_user_id` set if the "Include events from anonymous users" checkbox at the top of the data configuration section is selected.

Reviewer: ```suggestion
Next, select the message engagement events you would like to export. Reference the following export events and properties table listed. All events sent to Segment will include the user's `external_user_id` as the `userId` and the user's `braze_id` as the `anonymousId`.

Keep in mind, Braze only sends event data for users without an `external_user_id` if **Include events from anonymous users** is checked.
```

{% alert important %}

> Anonymous user export to Segment is currently in early access. Contact your Braze account manager if you’re interested in participating in this early access.

Reviewer: ```suggestion
Anonymous user export is currently in early access. Contact your Braze account manager if you’re interested in participating in this early access.
```
